gen/openapi/proto/rpc/system/idp.swagger.json

```diff
@@ -355,8 +355,7 @@
           "200": {
             "description": "A successful response.",
             "schema": {
-              "type": "object",
-              "properties": {}
```

Comment:
@nirav-rafay @akshay196 
docusaurus plugin was complaining empty `properties` i.e,
`"schema": { "type": "object", "properties": {} }` and it generates an invalid mdx file with no tabs. "schema": {} works as expected though.